src/Acquisition/_Acquisition.c

```diff
@@ -2030,7 +2109,22 @@ init_Acquisition(void)
   AcquisitionCAPI.AQ_Inner = capi_aq_inner;
   AcquisitionCAPI.AQ_Chain = capi_aq_chain;
 
-  api = PyCObject_FromVoidPtr(&AcquisitionCAPI, NULL);
+  api = PyCapsule_New(&AcquisitionCAPI, "AcquisitionCAPI", NULL);
```

Comment:
This macro relies on the following line. https://github.com/zopefoundation/Acquisition/blob/c-extension-py3/src/Acquisition/__init__.py#L941

So the `PyCapsule` object is added to the `Acquisition` module. If you have problems with the `aq_init()` macro it means that the import of `_Acquisition` fails. For testing you could remove the `try ... except ImportError` to see why.